flink-libraries/flink-table/src/main/scala/org/apache/flink/table/plan/schema/TableSourceTable.scala

```diff
@@ -25,6 +25,6 @@ import org.apache.flink.types.Row
 /** Table which defines an external table via a [[TableSource]] */
 class TableSourceTable(val tableSource: TableSource[_])
   extends FlinkTable[Row](
```

Comment:
I think it's not necessary to declare the return type is `Row`. `TableSource` could return Pojo in the future.

So it should be declared like `DataSetTable` and `DataStreamTable`, such as: 

```scala
class TableSourceTable[T](val tableSource: TableSource[T])
  extends FlinkTable[T](
```